src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -293,6 +294,38 @@ internal static string FindBestVersionMatch(VersionRange versionRange, IEnumerab
             return matchingVersion?.ToString();
         }
 
+        private NuGetVersion ResolvePlatformReleaseChannelVersion(IEnumerable<NuGetVersion> bundleVersions, NuGetVersion currentMatchingVersion)
+        {
+            bool isStandardOrExtendedChannel = string.Equals(_platformReleaseChannel, ScriptConstants.StandardPlatformChannelName, StringComparison.Ordinal) ||
```

Comment:
This data a case sensitive comparison? Should we do a case insensitive comparison to be safe?

Reply:
Or we could ToUpper or ToLower it